src/main/webapp/WEB-INF/tags/shared/feedbackResponseCommentRow.tag

```diff
@@ -27,7 +27,7 @@
 </c:choose>
 
 <li class="list-group-item list-group-item-warning" id="responseCommentRow-${divId}">
-    <div id="commentBar-${divId}">
+    <div id="commentBar-${divId}" style="display:inline;">
```

Comment:
If we're putting the issue of misaligned buttons to another issue, this change can probably be removed from this PR.